llvm/lib/CodeGenTypes/LowLevelType.cpp

```diff
@@ -22,6 +22,11 @@ LLT::LLT(MVT VT) {
     init(/*IsPointer=*/false, asVector, /*IsScalar=*/!asVector,
          VT.getVectorElementCount(), VT.getVectorElementType().getSizeInBits(),
          /*AddressSpace=*/0);
+  } else if (VT.isRISCVVectorTuple()) {
+    // TODO: Correctly model RISC-V vector tuple type
```

Comment:
Should we include this change? We don't have LLT support of tuples before, whether it is scalable or not.

Reply:
Current approach makes it use scalar construction, however once we change the TypeSize, `VT.getSizeInBits()` will no longer convertible directly to `size_t`.

Reply:
Setting IsScalar = true seems wrong thought it was the old behavior. What if we use the final else that sets all fields to `false` or `0`?

Reply:
A tuple is not a scalar. Why is setting IsScalar correct?

Reply:
I think after adding tuple type to `isScalableTargetExtVT`, we no longer need this lol